### DISCUSSION TITLE
Add Audio Player to linked Files as well

### DISCUSSION THREAD

**Discussion by fabian-hiller** [directus/directus#11627] [Feature Requests]
When you click on an audio file in the File Library, an audio player is displayed at the top.

<img width="852" alt="Screenshot 2022-02-14 at 10 55 50" src="https://user-images.githubusercontent.com/35291865/153899012-f1d08964-49bb-4c34-aa7e-32676ddea582.png">

However, when clicking on a linked audio file via an item and it opens on the right side of the screen on the same page, there is no audio player there. For us, this would greatly simplify the workflow if there was also an audio player there.

<img width="942" alt="Screenshot 2022-02-14 at 11 00 14" src="https://user-images.githubusercontent.com/35291865/153899655-17df7b0f-a607-4a29-bb35-5c06495805ff.png">

**Comment by benhaynes** [comment]
Yeah, I think we're adding all of the media previews to the drawers... @rijkvanzanten, isn't that in one of our current sprints? Or am I imagining that?

**Reply by rijkvanzanten** [reply]
That's correct!

**Reply by gpietro** [reply]
is it suppose to always appear? because when I upload the mp3 files I don't see it and would be great to have it.

**Reply by benhaynes** [reply]
As @rijkvanzanten confirmed, we have a PR (somewhere) that updates the file drawers to include the asset preview... not sure how close that is to being merged though.

**Reply by andy1li** [reply]
Audio player for mp3 files would be immensely useful! 🧡

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
